src/components/searchResults.tsx

```diff
@@ -0,0 +1,55 @@
+
+import { AccountDisplay } from '@src/components/accountDisplay';
+import { SearchResultsItem } from '@src/components/searchResultsItem';
+import { Account, AccountFields} from '@src/generated/sobs';
+import { Form } from 'antd';
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+
+interface SearchResultsProps {
+    accs: any;
+}
+
+interface SearchResultsStates {
+    selectedAccount: AccountFields;
```

Comment:
I think this state should be managed higher up in the component tree (related to below comment about how the search results component shouldn't be responsible for rendering the `AccountDisplay`)